snippets/rust/rust.json

```diff
@@ -91,12 +96,12 @@
     },
     "format": {
         "prefix": "format",
-        "body": ["format!(\"${1}\")"],
+        "body": ["format!(\"${1}\"${2})"],
```

Comment:
I thik that these kinds of macros that take `format_args!` are better if we do `format_args!("{}", ${1})` (properly escaped)